config/GregTech/GregTech.cfg

```diff
@@ -224,6 +223,45 @@ undergroundfluid {
         }
 
     }
+	
+	 nether {
+        # Dimension ID or Class Name
+        S:Dimension= -1
+		
+		lava {
+            # Chance generating (weighted chance!, there will be a fluid in chunk always!)
+            I:Chance=100
+
+            # Decrease per operation (actual fluid gained works like (Litre)VeinData/5000)
+            I:DecreasePerOperationAmount=1
+
+            # Max amount generation (per operation, sets the VeinData) 80000 MAX
+            I:MaxAmount= 50
```

Comment:
Why such a small amount?  If someone needs large quantities of lava and wants to get it from the Nether, this seems really low.